README.md

```diff
@@ -96,7 +96,15 @@ class MyHook
 }
 ```
 
-
+## Community Extensions
+
+* [Slack Gateway](presprog/contao-slack-notification) by [Present Progressive](https://www.presentprogressive.de): Send notifications to Slack channels or users.
```

Comment:
the URL looks wrong to me, shouldn't it also point to the extension list?

Reply:
Absolutely, yes, thank you